components/sharing-dialog/i18n/en.pot

```diff
@@ -5,107 +5,116 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2021-09-14T11:55:43.823Z\n"
-"PO-Revision-Date: 2021-09-14T11:55:43.823Z\n"
+"POT-Creation-Date: 2021-09-24T13:19:54.398Z\n"
+"PO-Revision-Date: 2021-09-24T13:19:54.398Z\n"
 
 msgid "Remove access"
 msgstr "Remove access"
 
 msgid "Not available offline"
 msgstr "Not available offline"
 
-msgid "Apply sharing to dashboard items"
-msgstr "Apply sharing to dashboard items"
+msgid "Apply sharing to dashboard visualizations"
+msgstr "Apply sharing to dashboard visualizations"
 
 msgid ""
-"All {{itemsCount}} items on this dashboard will be potentially updated with "
-"sharing settings from {{usersGroupsCount}} users and groups. Public access "
-"is not affected."
-msgstr ""
-"All {{itemsCount}} items on this dashboard will be potentially updated with "
-"sharing settings from {{usersGroupsCount}} users and groups. Public access "
-"is not affected."
+"{{count}} visualization on this dashboard will potentially get updated "
+"sharing settings."
+msgid_plural ""
+"{{count}} visualization on this dashboard will potentially get updated "
+"sharing settings."
+msgstr[0] ""
+"{{count}} visualization on this dashboard will potentially get updated "
+"sharing settings."
+msgstr[1] ""
+"All {{count}} visualizations on this dashboard will potentially get updated "
+"sharing settings."
 
-msgid ""
-"There aren't any sharing settings to apply to dashboard items. Public "
-"access cannot be applied to items."
-msgstr ""
-"There aren't any sharing settings to apply to dashboard items. Public "
-"access cannot be applied to items."
+msgid "These updated sharing settings will apply to {{count}} user or group."
+msgid_plural "These updated sharing settings will apply to {{count}} user or group."
+msgstr[0] "These updated sharing settings will apply to {{count}} user or group."
+msgstr[1] "These updated sharing settings will apply to {{count}} users and groups."
```

Comment:
I think this should be `users or groups`, rather than `users and groups`. The rest of the messages use _or_, and I can't see a reason to use _and_ here.